style.css

```diff
@@ -1,243 +1,316 @@
 @import url('https://fonts.googleapis.com/css?family=Poppins&display=swap');
 
+* {
+  box-sizing: border-box;
+}
+
 body {
   font-family: 'poppins', Arial, Helvetica, sans-serif;
   background-color: #e5e5e5;
-  margin-top: 40px;
-  margin-left: 20px;
-  width: 345px;
+  margin: 25px;
+  padding: 20px;
 }
 
-.header {
+@media all and (max-width: 1020) {
+  body {
+    padding: auto;
+    margin: auto;
+    width: auto;
+    height: auto;
+  }
+}
+
+.navbar {
   display: flex;
-  align-items: center;
   justify-content: space-between;
-  padding-top: 20px;
-  padding-right: 30px;
-  padding-left: 20px;
+  align-items: center;
+  color: black;
   background-color: #fff;
-  color: #6070ff;
+  padding-left: 100px;
+  padding-right: 100px;
+  padding-top: 20px;
+}
+
+.logo {
   font-weight: 700;
   font-size: 18px;
-  height: 20px;
+  color: #6070ff;
+  margin: 25px;
 }
 
-.head {
-  background-image: url(Assets/img/header-shapes.jpg);
+.nav-links ul {
+  margin: 0;
+  padding: 0;
   display: flex;
-  flex-direction: column;
-  margin-bottom: 90px;
-  padding-bottom: 100px;
-  border-bottom-left-radius: 100px;
 }
 
-.text {
-  display: flex;
+.nav-links li {
+  list-style: none;
+}
+
+.nav-links li a {
+  text-decoration: none;
+  color: black;
+  padding: 10px;
+  display: block;
+}
+
+.nav-links li :hover {
+  background-color: #6070ff;
+}
+
+.toggle-button {
+  top: 5px;
+  left: 5px;
+  display: none;
   flex-direction: column;
-  align-items: flex-start;
-  justify-content: flex-start;
-  padding-left: 20px;
-  padding-right: 30px;
-  padding-top: 100px;
+  justify-content: space-between;
+  width: 35px;
+  height: 25px;
 }
 
-.image1 {
-  display: grid;
-  grid-template-columns: auto;
-  gap: 5px;
-  background-color: #fff;
-  border-radius: 15px;
-  padding: 25px 25px 25px 20px;
-  width: 300px;
+.toggle-button .bar {
+  height: 4px;
+  width: 100%;
+  background-color: black;
+  border-radius: 7px;
+}
+
+@media all and (max-width: 1060px) {
+  .toggle-button {
+    display: flex;
+  }
+
+  .nav-links {
+    display: none;
+  }
+
+  .navbar {
+    width: 600px;
+  }
+}
+
+.header {
+  height: 50px;
+}
+
+.heading {
+  background-image: url(Assets/img/header.jpg);
+  border-bottom-left-radius: 150px;
   margin-bottom: 90px;
+  margin-top: 0;
 }
 
-.about {
-  background-color: #fff;
-  border-top-right-radius: 65px;
-  margin-left: 6px;
-  width: 335px;
+@media (max-width: 1060px) {
+  .heading {
+    width: 600px;
+    padding-right: 100px;
+    text-align: center;
```

Comment:
Great job so far👍. set header container to cover the whole page, without the grey background showing to meet up with [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) design
| Your Design |Figma Design |
| :---:   | :---: | 
|![uncover](https://user-images.githubusercontent.com/73854999/200120710-00f55ed7-339f-4ba5-bc94-85eb3601e82b.PNG)|![cover](https://user-images.githubusercontent.com/73854999/200120698-905837aa-53ce-441f-b958-abe4df3e26b4.PNG)|